File: src/main/java/org/radarbase/appserver/exception/EmailMessageTransmitException.java
package org.radarbase.appserver.exception;

public class EmailMessageTransmitException extends MessageTransmitException {
    public EmailMessageTransmitException(String message) {
        super(message);
    }
    public EmailMessageTransmitException(String message, Throwable e) {
        super(message, e);
    }
}

File: src/main/java/org/radarbase/appserver/exception/EmailMessageTransmitException.kt
package org.radarbase.appserver.exception

import java.io.Serial

@Suppress("unused")
class EmailMessageTransmitException : MessageTransmitException {

    companion object {
        @Serial
        private const val serialVersionUID = -1927189245766939L
    }

    constructor(message: String) : super(message)

    constructor(message: String, e: Throwable) : super(message, e)
}